strip-coverletter-docker.sh

```diff
@@ -1,24 +1,32 @@
 #!/bin/bash
 set -eu
 
-in=$(realpath $1)
-bname_in=${in##*/} # /foo/bar/baz.pdf = baz.pdf
+in=$(realpath "$1")
```

Comment:
I like the quotation marks good touch 👍

Reply:
Thanks to this brilliant tool: https://github.com/koalaman/shellcheck

I've been using it for years but this strip-coverletter script is some of my earliest bash work.